common/src/main/res/values/strings.xml

```diff
@@ -733,10 +737,14 @@
     <string name="shortcut5_note">Not all launchers support displaying 5 shortcuts, you may not see this shortcut if the above 4 are already added.</string>
     <string name="shortcuts_summary">Add shortcuts to launcher</string>
     <string name="shortcuts_tile">Shortcuts tile</string>
```

Comment:
This string is now unused and should be removed

Reply:
Removed in c3d81bf4